### ISSUE TITLE
Cannot add a comment containing an emoji when an LDAP user

### ISSUE THREAD

**Issue by phil-davis** [owncloud/user_ldap#279]
CI fails for:
```
--- Failed scenarios:

/var/www/owncloud/tests/acceptance/features/apiMetadataApps/comments.feature:16
``
which is testing an emoji in a comment.

Example drone run: https://drone.owncloud.com/owncloud/user_ldap/576/206

**Comment by phil-davis** [comment]
Adding QA team label so someone ( @individual-it ?) can first have a look at the test and confirm there is a real problem.

**Comment by individual-it** [comment]
works fine in a local test (automated test as well as manual test) - digging

**Comment by individual-it** [comment]
does not work with local drone run when using mysql but works when using sqlite
looking at the `config.php` it does not have `'mysql.utf8mb4' => true,` is the database in the docker container not setup correctly? @patrickjahns ideas?

**Comment by patrickjahns** [comment]
It's `image: mysql:5.5` - in core we use mysql:5.7 for utf8mb4 - ref: https://github.com/owncloud/core/blob/master/.drone.yml#L420-L422

**Comment by phil-davis** [comment]
Fixed by #281
